examples/typescript/actors/http-echo/README.md

```diff
@@ -0,0 +1,96 @@
+# Typescript HTTP Echo
+
+This repository
```

Comment:
Don't leave us in suspense!

Reply:
Oh hey I didn't know you were looking at this already! Check out the updated README!